client/js/widgets/seat.js

```diff
@@ -12,23 +12,27 @@ class Seat extends Widget {
       index: 1,
       turn: false,
       player: null,
-      display: null,
-      displayEmpty: null,
+      display: 'playerName',
+      displayEmpty: 'seatIndex',
       displayTurn: true,
       hideWhenUnused: false,
       hand: null,
 
       color: '#999999',
       colorEmpty: '#999999',
       layer: 1,
-      text: '1'
+      text:  'seatIndex'
     });
   }
 
   applyDeltaToDOM(delta) {
     super.applyDeltaToDOM(delta);
-    if(delta.text !== undefined)
-      setText(this.domElement, delta.text);
+    if((delta.text||delta.index||delta.player) !== undefined){
+      var displayedText = delta.text || this.get('text') ||""
```

Comment:
Any reason why this doesn't just do `this.get('text')` like other properties do?

Maybe just do `String(this.get('text')` because the next line should break if text is `1` (not `"1"`) for example.

Reply:
`this.get('text')` should give you `delta.text` if it is set.

Reply:
> `this.get('text')` should give you `delta.text` if it is set.

that sounds weird because that was the structure used in other places. there was nothing to indicate that both would result in the same thing